README.adoc

```diff
@@ -103,55 +103,36 @@ The code below assumes that your Docker registry is named `mydockerregistry`. Pl
 ----
 cd ./chat-ui
 
-nvm install
-nvm use
-npm install
-./protogen.sh
-npm run prestart
-npm run-script build
-
 # let's assume your DockerID in https://hub.docker.com/ is `mydockerregistry`
 DOCKER_PUBLISH_TO=mydockerregistry
 
 # build docker image
-docker build . -t $DOCKER_PUBLISH_TO/samples-js-chat-ui:latest
+docker build . -t $DOCKER_PUBLISH_TO/chat-ui-js:latest
```

Comment:
I suggest we name it just `chat-ui` since it is shared between all language-specific samples.
```suggestion
docker build . -t $DOCKER_PUBLISH_TO/chat-ui:latest
```

Reply:
Oh, I see. Would there be a case where there would be another Chat UI implemented in different language than JavaScript? If so, that `-js` suffix may still make sense.

Reply:
AFAIK there are no plans to create multiple UI implementations since Cloudstate is designed for creating backends. Moreover, this UI will be probably changed not to use a Cloudstate stateless service to deliver its contents.
And this UI is actually written mainly in TypeScript.